File: relationship-manager/conversation-service/src/main/java/com/bank/rm/conversation/ConversationServiceApplication.java
package com.bank.rm.conversation;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.bank.rm.conversation", "com.bank.rm.common"})
public class ConversationServiceApplication {
    public static void main(String[] args) {
        SpringApplication.run(ConversationServiceApplication.class, args);
    }
}

File: relationship-manager/conversation-service/src/main/kotlin/com/bank/rm/conversation/ConversationServiceApplication.kt
package com.bank.rm.conversation

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.conversation", "com.bank.rm.common"])
class ConversationServiceApplication

fun main(args: Array<String>) {
    runApplication<ConversationServiceApplication>(*args)
}
